libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -71,6 +102,11 @@ class AppPrompt {
       }));
     if (this.isAnchorExpanded) return;
 
+    const { base } = getConfig();
+
+    await loadStyle(`${base}/features/webapp-prompt/focus-animation.css`);
+    await loadStyle(`${base}/features/webapp-prompt/tooltip.css`);
```

Comment:
Was the performance impact of making two additional network requests compared to just concatenating these styles with the general PEP one?

Reply:
This is difficult to say. Networks have variance and are subject to long tails; singular or even tens of runs cannot adequately quantify which is faster because the question is really _how often_ is it faster. The answer to that question depends on the P75, P90, P99, (and so on) latency of our serverless environment. The answer to this question also depends on the user's device (and how hot it's running/how much battery it has, etc.) and the performance characteristics of brotli/gzip/zstandard decompression of files of different sizes.
A few cursory runs seem to indicate that the server should respond in under ~10ms, but this is subject to resource queueing delays in the browser, which are usually also under 10ms. But this is not data from the long tail.
If we want to optimize for the long tail (and I think we should), I would hazard a guess that we should probably put all the css into a single file -- the tradeoff is between project organization vs potential micro performance gain. 
I'd welcome more opinions on this.

Side note: More generally this question might be worth looking into with Milo more generally -- at what level of bundling vs splitting are we getting optimal performance (because we make many many blocking calls, and it takes only one P90 or worse latency to ruin the performance of the whole page; the probability of at least one P90 or worse latency in 30 network calls is 1 - 0.90^30 = ~96%. The probability that at least one will be P99 or worse is ~26%. 1 in 4 users would experience that)

Reply:
Personally, I think this is a bit of an overhead. If we were to conditionally load the focus/tooltip styles based on user interaction, then it would've made sense to separate. But since we need to load everything anyway, it just adds additional requests. Sure, that may be a very cost, but it adds up with time. I'd personally just visually separate the three areas in the main CSS file.

Not saying that this is a blocker, since PEP is loaded later on the page anyway, but might be worth considering.